test/micro/org/openjdk/bench/java/lang/StringBuffers.java

```diff
@@ -43,41 +43,10 @@
 @Fork(3)
 public class StringBuffers {
 
-    private String name;
-    private String blaha;
-    private Sigurd sig;
-
-    @Setup
-    public void setup() {
-        name = "joe";
-        blaha = "sniglogigloienlitenapasomarengrodasjukadjavelhej";
-        sig = new Sigurd();
-    }
-
-    @Benchmark
-    public String appendAndToString() {
-        return "MyStringBuffer named:" + ((name == null) ? "unknown" : name) + ".";
-    }
-
-    @Benchmark
-    public String toStringComplex() {
-        return sig.toString();
-    }
-
-    static class Sigurd {
-        int x;
-        byte y;
-        String z = "yahoo";
-
-        @Override
-        public String toString() {
-            return Integer.toString(x) + "_" + Integer.toString((int) y) + "_" + z + "_";
-        }
-    }
+    StringBuffer sb = new StringBuffer();
 
     @Benchmark
-    public String substring() {
-        return blaha.substring(30, 35);
+    public String emptyToString() {
```

Comment:
Do we actually need to remove the `substring` test here?

Reply:
I couldn't figure out why we'd want to have `String::substring` micros in a test `StringBuffers` (there's also a `StringSubstring` micro), though I could deal with that as an explicit cleanup RFE.

Reply:
Yeah, I would like to keep this change clean for backports. Do the cleanup in a separate PR?

Reply:
Reverted.